marklogic-data-hub-central/ui/src/components/queries/queries.tsx

```diff
@@ -639,27 +612,7 @@ const Query: React.FC<Props> = (props) => {
           }
 
         </div>}
-        <Modal
-          show={showEntityConfirmation}
```

Comment:
Is it ok the following step?
1. I create a query with an entity (can be any query with any entities)
2. I modify the sidebar to filter by more entities and other facets.
3. I click "Save query"
4. I expected a new query to be saved. No warning saving the previous one will be overwritten.
5. Query was overwritten when I wanted a new query saved. (As a dev/advanced user? I know there is a "save query as" option hidden in the menu. As a regular user I expected it to save a new query since I modified the name and description.

Reply:
@aniamy - Thanks for the feedback.
Given that the manage query behavior has been modified 5.7 onwards. Now whenever user chooses to update an existing query by changing base entities or facets, we have three options: -
1.  Save the new changes as part of current query (by using the Save icon).
2. Save it as part of a new query by using the "save as" option in the dropdown menu.
3. Revert the changes to go back to the last saved version of the query using "Revert query to saved State" option in the dropdown menu.
We have a plan to keep updating the save query behavior by making the new manage query options more predominant but those will be a part of separate tickets in future and we can definitely consider these suggestions as well.
@yubiselmarklogic - Just FYI.